src/lib/PreviewUI.js

```diff
@@ -264,9 +264,6 @@ class PreviewUI {
                 // part of finishLoadingSetup in BaseViewer.js
                 crawler.classList.remove(CLASS_HIDDEN);
             }
-
-            // Setup viewer notification
-            this.notification = new Notification(this.contentContainer);
         }
```

Comment:
Since you're in this file already, I'd move this check to be:

```
if (!this.contentContainer) {
    return;
}
```

to reduce arrow-ness